apps/research-service/src/site-polygons/site-polygons.controller.spec.ts

```
@@ -57,6 +60,22 @@ describe("SitePolygonsController", () => {
       await expect(controller.findMany({})).rejects.toThrow(UnauthorizedException);
     });
 
+    it("should throw an error if the countrselectedparams error is thrown", async () => {
```

Both tests has the same name : `should throw an error if the countrselectedparams error is thrown`

Also, the message description is not clear, I can't understand the behavior with the description. Can you please be more explicit?

---

Yes indeed, changed some of the names to the "should" pattern.